honeybee_radiance/cli/translate.py

```diff
@@ -0,0 +1,197 @@
+"""honeybee radiance translation commands."""
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click is not installed. Try `pip install . [cli]` command.'
+    )
+
+from honeybee_radiance.reader import string_to_dicts
+from honeybee_radiance.mutil import dict_to_modifier, modifier_class_from_type_string
+
+from ladybug.futil import write_to_file_by_name
+from honeybee.model import Model
+
+import sys
+import os
+import logging
+import json
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for translating Honeybee JSON files to/from RAD.')
+def translate():
+    pass
+
+@translate.command('model-to-rad-folder')
+@click.argument('model-json')
+@click.option('--folder', help='Folder on this computer, into which the Radiance '
+              'folders will be written. If None, the files will be output in the'
+              'same location as the model_json.', default=None, show_default=True)
+@click.option('--folder-type', help='An integer between 0-2 to note the type of '
+              'folders to be written out with the Model.\n0: grid-based\n1: view-based'
+              '\n2: includes both views and grids', default=2, show_default=True)
+@click.option('--minimal', help='Boolean to note whether the radiance strings should '
+              'be written in a minimal format (with spaces instead of line breaks).',
+              default=False, show_default=True)
+@click.option('--log-file', help='Optional log file to output the progress of the'
+              'translation. By default this will be printed out to stdout',
+              type=click.File('w'), default='-')
+def model_to_rad_folder(model_json, folder, folder_type, minimal, log_file):
```

Comment:
Should we also add config file as an optional input?

Reply:
Hm. This would need to be exposed as an option on the writer.model_to_rad_folder method first. I guess I can do this as part of this PR.

Reply:
Added it as an option.